script/preview.js

```diff
@@ -31,16 +34,28 @@ if (options.unexpected && options.unexpected.length !== 0) {
 }
 
 const app = express();
+const drupalClient = getDrupalClient(options);
 
 app.use(express.static(path.join(__dirname, '..', options.buildpath)));
 
-app.use('/preview', (req, res) => {
+app.use('/preview', async (req, res) => {
   const smith = createPipieline({
     ...options,
     port: process.env.PORT,
   });
 
   const contentId = req.query.contentId;
+  const pageData = await drupalClient.query({
+    query: GET_PAGE_BY_ID,
```

Comment:
I think the biggest reason behind why I prefer the query to be in its own file is that it's a different language, so it's a somewhat different way of thinking. If it were a shorter query, I could see adding them inline, but otherwise I think the consuming JS code should only need to know the exact function of the GraphQL query, and not the what the query itself looks like.